plugin/proto/errors.go

```diff
@@ -17,42 +17,40 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+
+	"github.com/notaryproject/notation-plugin-framework-go/plugin"
 )
 
-type ErrorCode string
+// Deprecated: ErrorCode exists for historical compatibility and should not be used.
+// To access ErrorCode, use the notation-plugin-framework-go's plugin.ErrorCode type.
+type ErrorCode = plugin.ErrorCode
 
 const (
 	// Any of the required request fields was empty,
 	// or a value was malformed/invalid.
-	ErrorCodeValidation ErrorCode = "VALIDATION_ERROR"
+	ErrorCodeValidation = plugin.ErrorCodeValidation
 
 	// The contract version used in the request is unsupported.
-	ErrorCodeUnsupportedContractVersion ErrorCode = "UNSUPPORTED_CONTRACT_VERSION"
+	ErrorCodeUnsupportedContractVersion = plugin.ErrorCodeUnsupportedContractVersion
 
 	// Authentication/authorization error to use given key.
-	ErrorCodeAccessDenied ErrorCode = "ACCESS_DENIED"
+	ErrorCodeAccessDenied = plugin.ErrorCodeAccessDenied
 
 	// The operation to generate signature timed out
 	// and can be retried by Notation.
-	ErrorCodeTimeout ErrorCode = "TIMEOUT"
+	ErrorCodeTimeout = plugin.ErrorCodeTimeout
 
 	// The operation to generate signature was throttles
 	// and can be retried by Notation.
-	ErrorCodeThrottled ErrorCode = "THROTTLED"
+	ErrorCodeThrottled = plugin.ErrorCodeThrottled
 
 	// Any general error that does not fall into any categories.
-	ErrorCodeGeneric ErrorCode = "ERROR"
+	ErrorCodeGeneric = plugin.ErrorCodeGeneric
 )
```

Comment:
Same here

Reply:
addressed in https://github.com/notaryproject/notation-go/pull/382